src/components/icons/staking/P2PGlyphIcon.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+import { createIcon } from "@chakra-ui/react"
+
+export const P2PGlyphIcon = createIcon({
+  displayName: "P2PGlyphIcon",
+  viewBox: "0 0 32 32",
+  defaultProps: {
+    width: "32px",
+    height: "32px",
+    fill: "currentColor",
+  },
+  path: [
+    <path
+      fill-rule="evenodd"
+      clip-rule="evenodd"
```

Comment:
Small syntax note... these attributes should be switched to camelCasing instead of kebab-casing to conform with React:
```suggestion
      fillRule="evenodd"
      clipRule="evenodd"
```